projects/demo/src/app/app.module.ts

```diff
@@ -1,19 +1,16 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, NO_ERRORS_SCHEMA } from '@angular/core';
-import { ReactiveFormsModule } from '@angular/forms';
+import { BrowserModule } from "@angular/platform-browser";
+import { NgModule, NO_ERRORS_SCHEMA } from "@angular/core";
+import { ReactiveFormsModule } from "@angular/forms";
 
-import { AppRoutingModule } from './app-routing.module';
-import { AppComponent } from './app.component';
-import { YearCalendarModule } from '../../../angular-year-calendar/src/public-api';
-import { ExampleSectionComponent } from './components/example-section/example-section.component';
-import { ConfigSideBarComponent } from './components/config-side-bar/config-side-bar.component';
+import { AppRoutingModule } from "./app-routing.module";
+import { AppComponent } from "./app.component";
+import { ExampleSectionComponent } from "./components/example-section/example-section.component";
+import { ConfigSideBarComponent } from "./components/config-side-bar/config-side-bar.component";
+import { YearCalendarModule } from "../../../angular-year-calendar/src/public-api";
+// import { YearCalendarModule } from '@iomechs/angular-year-calendar';
```

Comment:
I believe this needs to be reset? I.e. use the package instead of local public-api path?